schemas/collections.schemas.ts

```diff
@@ -22,6 +22,10 @@ export type Album = z.infer<typeof AlbumSchema>;
 export type Formats = z.infer<typeof FormatsSchema>;
 
 const SearchResponseSchema = z.object({
+  pagination: z.object({
```

Comment:
a common way of structuring pagination is like this:

```ts
{
  data: Album[]
  currentPage: number
  isLastPage: boolean
}
```

`pagination.pages` is confusing because it's not clear exactly what it is without having to dig through the code more